File: TMessagesProj/src/main/java/tw/nekomimi/nekogram/utils/EnvUtil.java
package tw.nekomimi.nekogram.utils;
import java.io.File;
import android.os.Environment;
public class EnvUtil {
    public static File getTelegramPath() {
        return new File(Environment.getExternalStorageDirectory(), "Telegram");
    }
    public static String[] getAvailableDirectories() { return new String[]{"", "", ""}; }
}

File: TMessagesProj/src/main/java/tw/nekomimi/nekogram/utils/EnvUtil.kt
package tw.nekomimi.nekogram.utils

import java.io.File

object EnvUtil {

    @JvmStatic
    val rootDirectories: List<File> by lazy { emptyList<File>() }

    @JvmStatic
    val availableDirectories
        get() = emptyArray<String>()

    @JvmStatic
    fun getTelegramPath(): File = File("")

    @JvmStatic
    fun getShareCachePath(): File = File("")

    @JvmStatic
    fun doTest() {}
}
